modules/apps/configuration-admin/configuration-admin-web/package.json

```diff
@@ -1,5 +1,6 @@
 {
 	"dependencies": {
+		"frontend-js-components-web": "*",
 		"frontend-js-web": "*"
```

Comment:
@ethib137 this is the only one that should be removed. All the other modules use `frontend-js-web`.

Reply:
Fixed (see push force below).